cmd/mailingTask.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"go_service/internal/common"
+	"log"
+	"time"
+)
+
+type SchedulerTimeGateway interface {
+	common.SchedulerReader
```

Comment:
As for me  `EmailSender, SubscriberListReader, CurrencyReader` need to be here too.